shell/platform/android/external_view_embedder/external_view_embedder.cc

```diff
@@ -57,6 +61,17 @@ bool AndroidExternalViewEmbedder::SubmitFrame(
 // |ExternalViewEmbedder|
 PostPrerollResult AndroidExternalViewEmbedder::PostPrerollAction(
     fml::RefPtr<fml::RasterThreadMerger> raster_thread_merger) {
+  bool has_platform_views = composition_order_.size() > 0;
```

Comment:
Does `composition_order_` represent the platform views in this frame or existing platform views.
For example, if you have 2 platform views on the screen, and this frame is going to remove both platform views making a total of 0 platform views on the screen after this frame. Is the size of `composition_order_` 0 or 2?
If it is 0, then we still need the threads to be merged. Because there are still 2 platform views on the screen and this frame is going to remove then, the removing process need to happen on the platform thread.

Reply:
Although having a kDefaultMergedLeaseDuration > 1 might already cover this case tho. Your call. On iOS, we sticked on the safe side to have the threads kept merged until we are sure there are no platform views both the previous frame and the current frame.

Reply:
I like that `kDefaultMergedLeaseDuration > 1` covers this case.

Reply:
Cool. maybe we can put a comment on it.

Reply:
Done in https://github.com/flutter/engine/pull/18841/commits/fb0f53b0b0716168b73483df8da5ca5815eb0af7.